actions/admin/access_keys.go

```diff
@@ -14,24 +15,24 @@ import (
 // @Description	Access Keys Search
 // @Tags		Admin
 // @Produce		json
-// @Param		SearchAccessKeys body SearchAccessKeys false "Supports targeted resource searches with filters and metadata, plus options for pagination and sorting to streamline data exploration and analysis"
+// @Param		SearchAccessKeys body filter.SearchAccessKeys false "Supports targeted resource searches with filters and metadata, plus options for pagination and sorting to streamline data exploration and analysis"
```

Comment:
Shouldn't it be `filter.AdminSearchAccessKeys`? (In swagger docs)

Reply:
Thanks 💯 
Fixed! 👍